spec/features/profiles_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'rails_helper'
+
+feature "zarządzanie profilem użytkownika" do
+  let(:email)           { "user@localhost.com" }
+  let(:real_password)   { "pass1234" }
+  let(:password)        { real_password }
+  
+  before :each do
+    Person.create!(name: "Anna", surname: "Genialna", email: email,
+                    sex: "kobieta")
+    User.create!(email: email, password: real_password, password_confirmation: real_password)
+  
+    visit '/users/sign_in'
+    within("#new_user") do
+      fill_in 'Adres e-mail', with: email
+      fill_in 'Hasło', with: password
+    end
+    click_button 'Zaloguj się'
+      
+    expect(page).to have_content 'pomyślnie'
+  end
+    
+  scenario "edytowanie profilu użytkownika" do
```

Comment:
W takim razie coś mi nie gra. Jak wspomniałam oparałam się na login_spec, a tam jest zrobione w ten sam sposób. Samo logowanie nie było umieszczone w osobnym scenariuszu, tylko wszystkie poprzedzało wykonując się dla każdego. Jeśli to nie jest dobrze tzn., że żaden test nie powinien mi zadziałać. Czy coś mi umknęło?